src/custom_type_glib_priority.rs

```diff
@@ -43,14 +43,16 @@ struct ReplaceToPriority {
 
 impl FunctionsMutVisitor for ReplaceToPriority {
     fn visit_function_mut(&mut self, func: &mut Function) -> bool {
-        if !func.name.ends_with("_async") {
```

Comment:
Sadly this change is not great because it ends up using the glib::Priority where it doesn't make sense.  Well there was only one case in gtk3-rs and another in gtk4-rs. E.g, https://docs.gtk.org/gtk3/method.TextTag.set_priority.html. Although, it drops a bunch of manual code from gtk4-rs, could be a fair trade, not sure :)

Reply:
will test it with gstreamer once the send on callbacks changes lands